src/DynamicDataTable.jsx

```diff
@@ -110,7 +123,7 @@ class DynamicDataTable extends Component {
         return (
             <div>
                 <div className="table-responsive">
-                    <table className="table table-striped">
+                    <table className={this.classNames()}>
```

Comment:
`this.classNames()` does not match the function definition on line 27. Should be `this.className()` or vise-versa.

Reply:
I've updated this 👍